src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/DependenciesGraphProvider.cs

```diff
@@ -142,18 +144,18 @@ public void BeginGetGraphData(IGraphContext context)
 
             Assumes.True(IsInitialized);
 
-            GraphNodeId newNodeId = GetTopLevelGraphNodeId(projectPath, viewModel.OriginalModel.GetTopLevelId());
+            GraphNodeId newNodeId = GetTopLevelGraphNodeId(projectPath, viewModel.OriginalModel!.GetTopLevelId());
```

Comment:
Asking for forgiveness with `!` here because `Requires.NotNull` is not annotated with `System.Runtime.CompilerServices.EnsuresNotNullAttribute`.